authbridge/demos/weather-agent/demo-ui.md

```
@@ -232,19 +227,22 @@ Wait for the Shipwright build to complete and the deployment to become ready.
 kubectl get pods -n team1
 ```
 
-Expected output:
+Expected output (cluster default — `proxy-sidecar` mode, no SPIRE):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)